TI-reports/2024/2024-Q4-BEST-WG.md

```diff
@@ -0,0 +1,225 @@
+
+
+### Python Hardening Guide
+
+#### Purpose
+
+- Help Python developers to create more secure code by explaining vulnerable and non-vulnerable coding patterns based on the CWE framework and rules.
+- Besides a description of each coding pattern, the guide includes executable code examples for each rule, which allow for an in-depth understanding of each pattern.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Scorecard
+
+#### Purpose
+
+- To help open source maintainers improve their security best practices and to help open source consumers judge whether their dependencies are safe.
+- Scorecard is an automated tool that assesses a number of important heuristics ("checks") associated with software security and assigns each check a score of 0-10.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Security Baseline
+
+#### Purpose
+
+- The goal of this SIG is to evolve OpenSSF security baseline for Linux Foundation wide adoption.
+- For OpenSSF adoption of the security baseline, there needs to be a home for tracking the adoption, for maintainers to raise issues to refine the security baseline, merge the baseline back to TAC lifecycle, and for OpenSSF to develop the roadmap for the security baseline. It will provide a venue for early adopters to share their reusable code and findings with other maintainers. The pilot adoption builds the foundation for wider adoption of the security baseline in OpenSSF and in Linux Foundation.
+- This SIG creates a venue for other participating foundations to help evolve the OpenSSF security baseline into a security baseline that can be applied to a broad range of software-based projects. The group will define the right level of risks that the security baseline is applicable for, the effectiveness measurement of the security baseline, and the adoption path of the security baseline at the minimum.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+### Web Developer Security Guide
+
+#### Purpose
+
+- Develop security best practice and guidelines specifically aimed at web developers.
+
+#### Current Status
+
+- We started the W3C SWAG group earlier this year and we've been holding weekly calls. The calls are minuted here: https://github.com/w3c-cg/swag/tree/main/meetings. We've so far produced an early draft security guidelines doc for web developers https://github.com/w3c-cg/swag/blob/main/docs/security_guidelines.md and we have another work item which is a guidelines doc for library developers. I've presented this work to the OpenJS Foundation security coord group as well to socialize it widely. We anticipate starting work on the libraries doc after the new year and as well continue to develop the web developer guide. Both documents will be at the level of detail of the "concise guide" document. We also held a special call where some Google security folks came to talk about tooling they are building around CSP and Trusted Types. Those talks will be posted shortly.
```

Comment:
Hi @torgo! Shall we move the second half of this paragraph to the "Up Next" section?

Reply:
Done.

Reply:
Thanks!

Reply:
@torgo can you fix the DCO issue?

Reply:
working on it...